news/2022-12-10-makeship-x-osu-plushies.md

```diff
@@ -0,0 +1,72 @@
+
+Feast your eyes on these glorious plushies in the form of *the* osu! cookie, pippi herself, and the much-loved Taikonator 3000:
+
+[![](/wiki/shared/news/2022-12-10-makeship-x-osu-plushies/launch.jpg)](https://www.makeship.com/search?q=PPY)
+
+Exciting, right? While we don't have a banana for scale, we do have an open bottle of authentic green tea, a headphone amplifier and a DAC instead. Can you guess whose desk this is? We bet you can.
+
+[![](/wiki/shared/news/2022-12-10-makeship-x-osu-plushies/banana-for-scale.jpg)](https://www.makeship.com/search?q=PPY)
+
+Something in this picture will make you go 'woot', and we suspect it probably isn't the keyboard.
+
+[![](/wiki/shared/news/2022-12-10-makeship-x-osu-plushies/mascots-keyboard.jpg)](https://www.makeship.com/search?q=PPY)
+
+Still not convinced? Witness a pippi plushie riding into the (osu! cookie) sunset atop the Taikonator 3000 as she leaves behind a hard life of farming scores for a new adventure that involves actually enjoying the game. She's crying because she's happy.
+
+[![](/wiki/shared/news/2022-12-10-makeship-x-osu-plushies/yeehaw.jpg)](https://www.makeship.com/search?q=PPY)
+
+Take a moment to look back on how far we've come, from clicking circles and making fan art to actually bringing both of them into the real world. In plushie form, no less.
+
+[![](/wiki/shared/news/2022-12-10-makeship-x-osu-plushies/the-og.jpg)](https://www.makeship.com/search?q=PPY)
+
+When our dashing developer [smoogipoo](https://osu.ppy.sh/users/1040328) isn't carefully considering code commits, he moonlights as a plushie salesman. Instead of punching time cards, our developers try to sell you things that just straight up punch you instead.
+
+[![](/wiki/shared/news/2022-12-10-makeship-x-osu-plushies/smoog.jpg)](https://www.makeship.com/search?q=PPY)
+
+Witness the Dean "peppy" Herbert waving his prototype osu! cookie in gentle disapproval at those who dare to leave the template visible in their fan art submissions.
+
+[![](/wiki/shared/news/2022-12-10-makeship-x-osu-plushies/waving-cookie.webp)](https://www.makeship.com/search?q=PPY)
+
+We also felt it was important to demonstrate just how throwable the osu! cookie plushie is. No reason.
+
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://assets.ppy.sh/media/makeship-x-osu/wang.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+If you like what you see, make them yours today by helping us fund the Makeship campaigns to bring them into reality. But get in quick, because these plushies are a **limited edition** run - so if you don't buy them this time, they're probably not coming back again!
```

Comment:
```suggestion
If you like what you see, make them yours today by helping us fund the Makeship campaigns to bring them into reality. But get in quick, because these plushies are a **limited-edition** run — so if you don't buy them this time, they're probably not coming back again!
```